### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by A-RANG3R**
This contains the initial repository structure for the game. We have a basic game set up with a few basic commands. Some of the base files may be added or removed, but it should generally look about like this.

Changes to pay attention to: any .java file.

### CODE REVIEW

**Comment by A-RANG3R** [comment]
added no args constructors to all non-main classes

**Comment by dwahme** [comment]
@blun0te @MuffoloMan don't approve this request until you're successfully able to "play" on your local machine.

File: src/cli/CLI.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    //No args constructor
+    public CLI() 
+    {
+
+    }
+
+    // Returns if we need to exit the game
+    public Boolean endGame()
+    {
+        return exit;
```
Comment (MuffoloMan) [inline src/cli/CLI.java]: Why is this called exit, it should be something better.

Comment (A-RANG3R) [reply src/cli/CLI.java]: Changed to endGame

**Pull request merged**